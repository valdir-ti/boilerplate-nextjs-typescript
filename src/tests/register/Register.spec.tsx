import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'

import Register from '../../pages/register'

describe('<Register />', () => {
  it('should render a heading', () => {
    render(<Register />)
    expect(
      screen.getByRole('heading', { name: /Register/i })
    ).toBeInTheDocument()
  })

  it('should match a snapshot', () => {
    const { container } = render(<Register />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
