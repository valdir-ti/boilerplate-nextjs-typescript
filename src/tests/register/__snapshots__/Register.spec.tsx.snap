// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Register /> should match a snapshot 1`] = `
<div>
  <h2>
    Register
  </h2>
</div>
`;
